data_storage/export_format.md

```diff
@@ -1,22 +1,28 @@
 Export Format
 -------------
 
-- **5.1.1** ![](/badge/req.png) A Toxcore data file saved in the root of the client's
-  working directory, with the file name `<profile_name>.tox`
+- **5.1.1** ![](/badge/req.png) The client will export files relative to a folder called
+  the export folder, which must be in a location accessible to the user.
 
-- **5.1.2** ![](/badge/req.png) If the client supports user avatars, they will be stored
-  under the `avatars/` directory in the root of the client's working directory.
+- **5.1.2** ![](/badge/rec.png) The user should be able to choose the location
+  of the export folder.
 
-- **5.1.3** ![](/badge/req.png) Avatars are named according to the following format:
+- **5.1.3** ![](/badge/req.png) A Toxcore data file saved in the root of the
+  export folder, with the file name `<profile_name>.tox`.
```

Comment:
I think we disagree on what is toxcore's job. I'm on the far end of "should provide networking facilities only". Ideally, it does exactly that and specialises on it, while other libraries built on top can provide default implementations of standardised behaviour (i.e. things in TCS and related documents).

Indeed not everything that's in the tox save data can be extracted, but it's possible to get a tox instance to its original state (modulo connected DHT nodes) using API calls and not using save data. I would like to see the save data going away or using a format I can easily inspect and modify from code without manually writing specialised parsers and generators for it. However, that discussion doesn't belong here, so given the current state of toxcore, we might as well make use of what's there. The only thing that worries me is that if we agree to use the blob created by toxcore, any change to that decision will create a backwards-incompatible change in the standard.